### DISCUSSION TITLE
Beginner's Question: Diagnosis at the time of hospitalization

### DISCUSSION THREAD

**Discussion by sometion** [MIT-LCP/mimic-code#1715] [MIMIC-IV]
I am very sorry for the basic question.

I would like to identify a subject who has been diagnosed with a certain complication and does not have that diagnosis at the time of admission.

The diagnoses_icd does not identify the date, so I do not know if the disease was present at the time of admission. admissions does not have an icd, so I am not sure if it was present at the time of admission.

Which resource should I use to identify the person's history and the disease at the time of admission?

**Comment by adviksh** [comment]
Some of this might be in the `chiefcomplaint` in the `triage` table for [MIMIC-IV-ED](https://physionet.org/content/mimic-iv-ed/2.2/). You may also be able to piece together what some patients presented with based on discharge summaries from [MIMIC-IV-Note](https://physionet.org/content/mimic-iv-note/2.2/).
 
I don't know if you can determine exactly what conditions every patient was known to have at the time of admission, so there will likely be missing data problems with either of the approaches above.

I've been thinking about this as well, and would be curious if you figure out a solution!

**Reply by alistairewj** [reply]
Admission diagnosis is notoriously difficult. Unfortunately it's just not documented anywhere consistently. The above are good suggestions. Here are my suggestions.

1. `chiefcomplaint` from [MIMIC-IV-ED](https://physionet.org/content/mimic-iv-ed/), as mentioned, is written by the nurse on triage. It's free text but due to some nice auto-complete at BIDMC the quality is fairly good and it's reasonably structured.
2. The *diagnosis* table in [MIMIC-IV-ED](https://physionet.org/content/mimic-iv-ed/) has diagnoses billed on discharge from the ED. Strictly speaking, this has the same issues as the hospital based diagnoses, in that they are retrospective. For this reason you can see some substantial differences: for example `stay_id` 31115595 has a chief complaint of "wound eval" and an ICD diagnosis of sepsis. `stay_id` 33380565 has status post fall and an ICD diagnosis of head contusion, etc. For many though, it's consistent (chief complaint of abd pain, and ED discharge diagnosis of abd pain).
3. The discharge summary in the *discharge* table of [MIMIC-IV-Note](https://physionet.org/content/mimic-iv-note) has the standard fields written in an admission note: history of present illness, past medical history, etc. Unfortunately this note is written on discharge (with the benefit of hind sight), but you can also use that section. It's usually free-form so deriving structured admission diagnoses from this field would be hard.
4. If you are focusing on ICU patients, then there is a field added in [MIMIC-IV v2.0 (and available in v2.2)](https://physionet.org/content/mimiciv/2.2/#release-notes) with the active problem. This isn't exactly an admission diagnosis, it's the set of problems being managed at the time, but if you filter this to within the first 6-12 hours or so it often gives you a pretty clear idea of what the admitting issue is. The benefit is that it's a structured field (around ~1000 unique possible entries).
```sql
SELECT c.stay_id
, c.charttime
, c.value
FROM `physionet-data.mimiciv_icu.chartevents` c
WHERE c.itemid = 220001
LIMIT 100
```
Gives values such as:
* .Care Plan - Altered Respiratory Function: Ineffective Breathing Patterns
* "Pulmonary Embolism (PE), Acute"
* .Care Plan - Alteration in Sensory / Perception
* ".H/O gastrointestinal bleed, lower (Hematochezia, BRBPR, GI Bleed, GIB)"
* "Balance, Impaired"
